internal/resource/config/sp/idpconnection/sp_idp_connection_resource.go

```diff
@@ -0,0 +1,4529 @@
+											},
+											"value": schema.StringAttribute{
+												Computed:            true,
+												Optional:            true,
+												Description:         "The value for this attribute.",
+												MarkdownDescription: "The value for this attribute.",
+												Default:             stringdefault.StaticString(""),
+											},
+										},
+									},
+									Required:            true,
+									Description:         "A list of mappings from attribute names to their fulfillment values.",
+									MarkdownDescription: "A list of mappings from attribute names to their fulfillment values.",
+								},
+								"attribute_sources": attributesources.ToSchema(0, false),
+								"authentication_policy_contract_ref": schema.SingleNestedAttribute{
+									Attributes:          resourcelink.ToSchema(),
+									Required:            true,
+									Description:         "A reference to a resource.",
+									MarkdownDescription: "A reference to a resource.",
+								},
+								"issuance_criteria": schema.SingleNestedAttribute{
```

Comment:
This could also use the common package

Reply:
No default value on this one.

Reply:
I'm surprised they wouldn't be able to use the same default, since it's just defaulting conditional_criteria to an empty set.